src/test/java/org/zalando/pazuzu/ResolvedFeatureApiTest.java

```diff
@@ -0,0 +1,71 @@
+package org.zalando.pazuzu;
+
+import org.junit.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ResolvedFeatureApiTest extends AbstractComponentTest {
```

Comment:
there is no empty requests here the name parameter is required you can not resolve all features, and recursive dependency are prevented during the create & update methods so we can't create fixture data to test it.

Will implement feature not found test